### ISSUE TITLE
How I can remove old translations which was already deleted from code?

### ISSUE THREAD

**Issue by max-mykhailenko** [i18next/i18next-scanner#30]
I already try to use loadPath only as separate file. My script:

```node
const scanner = require('i18next-scanner');
const languages = require('./languages.json');
const vfs = require('vinyl-fs');
const fs = require('fs');
const path = require('path');

Object.keys(languages).forEach((k) => {
  const fromFile = path.resolve(__dirname, `../locales/${languages[k]}/translation.json`);
  const toFile = path.resolve(__dirname, `../locales/${languages[k]}/translation-prev.json`);
  fs.renameSync(fromFile, toFile);
});

const options = {
  attr: false,
  func: {
    list: ['_', 'trans'],
    extensions: ['.js', '.jsx', '.php', '.twig'],
  },
  lngs: Object.keys(languages).map(k => languages[k]),
  ns: 'translation',
  defaultNs: 'translation',
  resource: {
    loadPath: 'locales/{{lng}}/{{ns}}-prev.json',
    savePath: '{{lng}}/{{ns}}.json',
  },
  nsSeparator: false,
  keySeparator: false,
};

vfs.src([
  'source/js/**/*.js'
])
.pipe(scanner(options))
.pipe(vfs.dest('locales'));

```

**Comment by cheton** [comment]
Do you have translation keys that cannot be scanned from code and need to maintain manually?

I used to manage resource files by namespace, for example:
* `translation.json` will always be scanned from code
* `locale.json` is manually maintained

The `translation.json` file will be removed prior to scanning, so it won't keep old translations that was already deleted from code.

**Comment by max-mykhailenko** [comment]
Nope, I don't need additional translation keys and want to left only keys from code.

**Comment by cheton** [comment]
Hmm...I can provide an option to remove old translation keys that's already deleted. I will work on it in the next few days.

**Comment by cheton** [comment]
A new option `removeUnusedKeys` is now available in v1.3.0.  You can set it to `true` to remove unused translation keys.

```js
const options = {
  removeUnusedKeys: true,
  attr: false,
  func: {
    list: ['_', 'trans'],
    extensions: ['.js', '.jsx', '.php', '.twig'],
  },
  lngs: Object.keys(languages).map(k => languages[k]),
  ns: 'translation',
  defaultNs: 'translation',
  resource: {
    loadPath: 'locales/{{lng}}/{{ns}}-prev.json',
    savePath: '{{lng}}/{{ns}}.json',
  },
  nsSeparator: false,
  keySeparator: false,
};
```

**Comment by max-mykhailenko** [comment]
You are cool! Thanks!
